aikau/src/main/resources/alfresco/services/ActionService.js

```diff
@@ -385,7 +385,7 @@ define(["dojo/_base/declare",
        *
        * @instance
        */
-      onSelectedFilesChanged: function alfresco_services_ActionService__onSelectedFilesChanged() {
+      onselectedItemsChanged: function alfresco_services_ActionService__onselectedItemsChanged() {
```

Comment:
Is there a reason for this having a lower-case `s`? Am assuming there is, as it looks deliberate.